src/apps/ENGINE/file_service.cpp

```diff
@@ -133,6 +133,10 @@ std::vector<std::filesystem::path> FILE_SERVICE::_GetFsPathsByMask(const char *s
     auto it = std::filesystem::directory_iterator(srcPath, ec);
     if (ec)
     {
+        if (core.tracelog)
+        {
+            core.tracelog->warn("Failed to open save folder: {}", ec.message());
```

Comment:
thanks for the change
but generally, GetFsPathsByMask can be used not only for querying the save folder
it would be better to look like "failed to get directory_iterator for {}", srcPath